src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -11,6 +11,7 @@
 import dateutil.parser
 
 from azure.cli.core._util import CLIError, todict, get_file_json
+from azure.cli.core._profile import Profile
```

Comment:
For perf. reasons, can we move this import to inside the method it's used in.

importing Profile was v. slow in my experience.